src/WalterModem.h

```diff
@@ -229,6 +229,7 @@
  */
 #define SPI_FLASH_BLOCK_SIZE    (SPI_SECTORS_PER_BLOCK*SPI_FLASH_SEC_SIZE)
 
+#define WALTER_MODEM_MAX_CONNECTION_EVENT_HANDLERS 6
```

Comment:
Please comment the define and I would propose to rename to `WALTER_MODEM_MAX_EVENT_HANDLERS` and keep a single array of all types of event handlers.

```

#define WALTER_MODEM_MAX_EVENT_HANDLERS 6

typedef enum {
    WALTER_MODEM_EVENT_GROUP_CONNECTION,
    WALTER_MODEM_EVENT_GROUP_AT,
} WalterModemEventGroup;

typedef struct {
 WalterModemEventGroup group
 void* handler;
} WalterModemEventHandler;

static inline WalterModemEventHandler _eventHandlers[WALTER_MODEM_MAX_EVENT_HANDLERS ];
```